sprocs/assessment_questions_calculate_stats.sql

```diff
@@ -75,7 +75,7 @@ aq_stats AS (
     SELECT
         least(100, greatest(0, avg(question_stats_by_user_or_group.score_perc)))                     AS mean_question_score,
         sqrt(var_pop(question_stats_by_user_or_group.score_perc))                                    AS question_score_variance,
-        corr(question_stats_by_user_or_group.score_perc, assessment_scores_by_user_or_group.score_perc) * 100 AS discrimination,
+        coalesce(corr(question_stats_by_user_or_group.score_perc, assessment_scores_by_user_or_group.score_perc) * 100, 0) AS discrimination,
```

Comment:
There is a minor issue with this approach, which is the fact that, if there are no submissions, all other stats will be null, but the discrimination will be 0. This is by no means major, and I'd be fine if you were to ignore this case, but if consistency is to be considered it may be a good idea to have some way to address that, maybe like this?
```suggestion
        coalesce(corr(question_stats_by_user_or_group.score_perc, assessment_scores_by_user_or_group.score_perc) * 100, CASE WHEN count(question_stats_by_user_or_group.score_perc) > 0 THEN 0 ELSE NULL END) AS discrimination,
```

Reply:
I'm on board with this.